Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_5_Word Embedding_(D).md

```diff
@@ -0,0 +1,138 @@
+## Title : Name of the Topic
```

Comment:
title : Name of the topic!

But where is that 'name of the topic' you're discussing here. The typo needs to be fixed.